docs/sce_file_contents.md

```diff
@@ -170,32 +177,32 @@ The list of highly variable genes used was selected using `scran::modelGeneVar`
 The following command can be used to access the PCA results:
 
 ```r
-reducedDim(sce, "PCA")
+pca <- reducedDim(sce, "PCA")
 ```
 
 UMAP results were calculated using `scater::runUMAP()`, with the PCA results as input rather than the full gene expression matrix.
 The following command can be used to access the UMAP results:
 
 ```r
-reducedDim(sce,"UMAP")
+umap <- reducedDim(sce,"UMAP")
```

Comment:
Noting missing space, but I would lose the `umap <- ` as well per earlier comments.
```suggestion
umap <- reducedDim(sce, "UMAP")
```